### DISCUSSION TITLE
Audiobookshelf Listening Sessions

### DISCUSSION THREAD

**Discussion by arcimus** [LeoKlaus/plappa#15] [Feature Requests]
Playback of downloaded audiobooks from Audiobookshelf works great.  Completed books are properly marked as finished in Audiobookshelf.  Progress is synced with the server to allow to continue listening on the website.  However, it appears that Listening Sessions are not created, which affects some real-time and recent reporting.

**Comment by LeoKlaus** [comment]
That is correct. I've already started working on this but have scratched it for now as I ran into some issues with implementing the syncing process (leading to wrong stats in ABS, which imo is worse than no stats at all).

I think this would be much easier to implement if plappa started tracking sessions instead of just progress, so I might be able to close this with #14.

**Reply by mutschler** [reply]
well it seems like there is some server side processing for that in place already(at least that's what i take away from reading https://github.com/advplyr/audiobookshelf-app/issues/1076#issuecomment-1974904248) might be a good startingpoint for implementing this in plappa :)

**Comment by justcallmelarry** [comment]
love the look of the app, basically waiting for stats in order to try it out 😄

**Comment by LeoKlaus** [comment]
I'm closing this discussion as there is a feature request at #83.
